aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/includes/question_list.html

```diff
@@ -12,10 +12,20 @@ <h3 class="panel-title">{{ qgroup.grouper }}</h3>
         {% endif %}
         <div class="list-group">
             {% for question in qgroup.list %}
-                <a href="{{ question.get_absolute_url }}" class="list-group-item">
-                    {% render_model question "title" %}
-                    {% if not list %}<span class="badge">{{ qgroup.grouper }}</span>{% endif %}
-                </a>
+                {% if view.config.app_data.config.show_description %}
+                    <div class="list-group-item">
+                        <a href="{{ question.get_absolute_url }}">
+                            {% render_model question "title" %}
+                            {% if not list %}<span class="badge">{{ qgroup.grouper }}</span>{% endif %}
+                        </a>
+                        {% render_model question "answer_text" %}
```

Comment:
shouldn't this also be clickable?
like in this example http://getbootstrap.com/components/#list-group-custom-content